corehq/apps/toggle_ui/utils.py

```diff
@@ -1,7 +1,106 @@
+from couchexport.writers import Excel2007ExportWriter
+from django.utils.translation import ugettext as _
+from io import BytesIO
 from corehq.toggles import all_toggles
 
+from corehq.apps.accounting.models import Subscription
+from corehq.apps.es import UserES
+
 
 def find_static_toggle(slug):
     for toggle in all_toggles():
         if toggle.slug == slug:
             return toggle
+
+
+def get_subscription_info(domain):
```

Comment:
should cache this

Reply:
Caching increased the export with 46%...nice!